src/engine/controls/cuecontrol.cpp

```diff
@@ -752,7 +825,92 @@ void CueControl::hotcueGotoAndPlay(HotcueControl* pControl, double v) {
     }
 }
 
-void CueControl::hotcueActivate(HotcueControl* pControl, double v) {
+void CueControl::hotcueGotoAndLoop(HotcueControl* pControl, double v) {
+    if (!v)
+        return;
```

Comment:
This could potentially break some controller scripts that set it to -1 or something like that. Not sure if that is worth it. I'll add braces throughout the file though.

Reply:
If controller scripts are setting buttons to anything other than 0 and 1 they're doing it wrong.